### PR TITLE
Updated the read me file 

### LINKED ISSUE #3: Updating the read me file
The content in the readme file needs to be updated.

### PR DESCRIPTION
**Pull request by Mohameddekow**
This fixes #3

### CODE REVIEW

**Comment by Ericgacoki** [comment]
Hey @Mohameddekow, your PR will be reviewed ASAP before it's merged. Thanks ☺️

**Review by Ericgacoki — APPROVED** [review]
good to go 😊

**Comment by Ericgacoki** [comment]
Ready to merge.

File: README.md [hunk 1]
```diff
@@ -13,4 +13,4 @@ This project implements the following `Android` concepts:
 - `RecyclerView`
 - `ImgeVIew` 
 -->
-... documentation in progress 🔥🔥
```
Comment (aureliamutua) [inline README.md:16]: ..... documentation in progress

**Pull request merged**